en/core-libraries/httpclient.rst

```diff
@@ -479,6 +479,59 @@ instead. You can force select a transport adapter using a constructor option::
 
     $client = new Client(['adapter' => Stream::class]);
 
+Events
+======
+
+The HTTP client triggers couple of events before and after sending a request
+which allows you to modify either the request or response or do other tasks like
+caching, logging etc.
+
+HttpClient.beforeSend
+---------------------
+
+    // Somewhere before calling one of the HTTP client's methods which makes a request
+    $http->getEventManager()->on(
```

Comment:
```suggestion
    $client->getEventManager()->on(
```

Reply:
All code examples above use the var `$http` for the client instance, hence I used the same for consistency.

Reply:
The example on 480 uses $client. I can take care of normalizing them all though.